### DISCUSSION TITLE
Unwanted i3 icons on status bar

### DISCUSSION THREAD

**Discussion by nadav7679** [i3/i3#5674] [Q&A]
Hi,
I am running Ubuntu 22.04 and I installed i3 and Polybar.
Ever since I installed i3, I have these annoying three icons on the statusbar that I can't get rid of them. I deleted the whole "bar" portion of the config file, and even installed Polybar, but these icons keep popping up. They are always on the right most side (before the Polybar modules), and they move accordingly to the bar placement (top or bottom). 
Also, the Polybar transparency settings change for me. Even though I did not define transparency, the bar has a transparent portion.
I couldn't find any info about it online, so I would love some help. 

![status icons](https://github.com/i3/i3/assets/65134110/2d059c93-e49a-4f31-bb20-15c34c05fbe9)

**Comment by orestisfl** [comment]
These are not configured by i3, they are applications that are running and
have a tray icon. Check which applications are executed on startup and try
removing them.
